src/main/java/com/atlassian/db/replica/api/reason/Reason.java

```diff
@@ -55,6 +63,35 @@ public String toString() {
         return "Reason{" +
             "name='" + name + '\'' +
             ", isRunOnMain=" + isRunOnMain +
+            ", isWrite=" + isWrite +
             '}';
     }
+
+    private static class ReasonBuilder {
+        private String name;
+        private boolean isRunOnMain;
```

Comment:
Can you make the default values explicit?

Reply:
fixed